README.md

```diff
@@ -147,7 +147,10 @@ conda env create -f environment.yml
 ```
 
 ## Panel format
-Many of the scripts in `toffy` require a panel file. This file identifies which targets have been put on which masses. For an example of what the format should be, you can look at the [example panel file](https://github.com/angelolab/toffy/blob/main/files/example_panel_file.csv). Some panels will not have targets on every mass; in this case, it's important that you just leave the placeholder row in the panel, and not delete it, in order to ensure that all the notebooks work as expected. Similarly, if you have multiple targets on the same mass, don't add a unique row for each, just give them a consolidated name. 
+Many of the scripts in `toffy` require a panel file, identifying which targets have been put on which masses.  
+You can download your panel [here](https://angelolab.ionpath.com/tracker/panel); select the correct panel and click `Download csv`.
```

Comment:
This should instead be a screenshot of the panel interface on the tracker and where the download button is. Other labs will have their own link, not angelolab.